powerauth-push-client/pom.xml

```diff
@@ -21,32 +21,13 @@
             <artifactId>powerauth-push-model</artifactId>
         </dependency>
 
-        <!-- Spring Dependencies -->
+        <!-- Wultra dependencies -->
         <dependency>
-            <groupId>org.springframework.boot</groupId>
```

Comment:
It does not make sense to force clients to use Spring Boot. It compiles without these dependencies.